content/en/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md

```diff
@@ -258,15 +258,26 @@ include it.
 In addition to enabling the feature in the cluster, a resource driver also has to
 be installed. Please refer to the driver's documentation for details.
 
-### Enabling Device Status
+## Devices status
 
-[ResourceClaim Device Status](#resourceclaim-device-status) is an *alpha feature* 
-and only enabled when the `DRAResourceClaimDeviceStatus` 
-[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
-is enabled in the kube-apiserver.
+{{< feature-state feature_gate_name="ResourceHealthStatus" >}}
+
+There are cases when devices fail or are shut down. The responsibility of the DRA plugin
+in this case is to notify the kubelet about the situation using the `WatchResources` API.
+Code that relies on the devices will usually fail. Pods that don't set the `restartPolicy`
+field to `Always` might enter the `Failed` phase. Pods that do set the `restartPolicy`
+field to `Always` might enter a crash loop.
+
+If you enable the `ResourceHealthStatus` feature gate, the `allocatedResourcesStatus` field
+is added to each entry in the `status.containerStatuses` field of the Pod specification. The `allocatedResourcesStatus`
+field reports health information for each device assigned to the container.
```

Comment:
Putting this as-is into the 1.32 is IMHO misleading the users.

First, the part which gets health information from a DRA driver and publishes that in the pod status hasn't actually been implemented in kubelet, if I am not mistaken.

Second, even if it was, a DRA driver has to support this new feature. This needs to be mentioned in this documentation here, otherwise users may look for this promised information and then be surprised that it's not there.

Reply:
so do we need an update for 1.32?